src/services/machines/funkeC2ShareMachineService.ts

```diff
@@ -240,9 +270,33 @@ export const storePIDCredentials = async (context: Pick<FunkeC2ShareMachineConte
       },
       opts: {hasher: generateDigest},
     });
+
     if (!parentId) {
       parentId = digitalCredential.id;
+      parentCredentialHash = digitalCredential.hash;
     }
+
+    store.dispatch<any>(
+      storeActivityLogging({
```

Comment:
That would be an audit event not an activity

Reply:
changed